packages/@rnw-scripts/babel-react-native-config/babel.config.js

```diff
@@ -11,7 +11,6 @@ module.exports = () => ({
     'module:@react-native/babel-preset',
   ],
   plugins: [
-    "babel-plugin-syntax-hermes-parser",
```

Comment:
we still need this until the jest-preset fix is published

Reply:
This plugin was added to `@react-native/babel-preset` in https://github.com/facebook/react-native/commit/1387f521fdd8f187eab7a4a6a05d4d75a96b4f88. We should be able to remove this line if we also bump the preset to 0.77-nightly: https://github.com/TatianaKapos/react-native-windows/blob/a459663cd72ab4d151915bb19461847a9f26ea70/packages/%40rnw-scripts/babel-react-native-config/package.json#L14-L19

Reply:
that seemed to fix it! Thanks Tommy! :)